velox/exec/AggregateCompanionAdapter.cpp

```diff
@@ -261,14 +261,14 @@ bool CompanionFunctionsRegistrar::registerPartialFunction(
              CompanionSignatures::partialFunctionName(name),
              std::move(partialSignatures),
              [name](
-                 core::AggregationNode::Step step,
+                 core::AggregationNode::Aggregate::Step step,
                  const std::vector<TypePtr>& argTypes,
                  const TypePtr& resultType,
                  const core::QueryConfig& config)
                  -> std::unique_ptr<Aggregate> {
                if (auto func = getAggregateFunctionEntry(name)) {
                  if (!exec::isRawInput(step)) {
-                   step = core::AggregationNode::Step::kIntermediate;
+                   step = core::AggregationNode::Aggregate::Step::kIntermediate;
```

Comment:
@zhztheplayer 

It is a bit difficult to see the material changes because of all the renaming of core::AggregationNode::Step to core::AggregationNode::Aggregate::Step. 

Would it be possible to create a version of this PR that doesn't do the rename, just moves core::AggregationNode::Step property from the plan node to Aggregate?

Reply:
@mbasmanova 

That would be possible. But I'd clarify would you use that PR just for reviewing or you actually prefer that approach?

Reply:
Thanks. At this point, I'm still trying to understand the proposal and the changes. I do not have a clear preference yet. Hope this makes sense.

Reply:
Yes I saw your latest comments in https://github.com/facebookincubator/velox/issues/12830. I'll follow up on that.